c10/util/Optional.h

```diff
@@ -476,7 +476,19 @@ class optional : private OptionalBase<T> {
 
   optional(const optional& rhs) = default;
 
+// CUDA 9.2 and below doesn't compile default move constructor correctly
```

Comment:
Can you please add a reference to the issue here?
```suggestion
// CUDA 9.2 and below fail while trying to compile default move constructor
```